<launch>
	<arg name="topic" default="/kinect2/qhd/points"/>
	<include file="$(find pcl_tracker)/launch/ur5_bringup.launch"> 
		<arg name="show_rviz" value="true"/>
	</include>
	<include file="$(find fingertip_detector)/launch/fingertip_detector.launch"/>
	<include file="$(find move_ur5_qt)/launch/move_ur5_qt.launch"/>
	<!--include file="$(find cnn_hgr)/launch/HGR.launch"/-->
</launch>
